content/_data/upgrades/2-346-1.adoc

```diff
@@ -0,0 +1,46 @@
+==== PNG and GIF icons replaced with SVG icons
+
+Scalable vector graphics images are used in Jenkins instead of the GIF and PNG images that were used previously.
+SVG images provide a better user experience and improve the Jenkins user experience.
+
+The GIF and PNG images have been removed from Jenkins core.
+
+Jenkins has removed a large number of GIF and PNG iamges.
```

Comment:
Redundant

Reply:
(Also, typo: iamges)

Reply:
Removed in ae79c36ac49fcb44c5084649bf95f721a2c99aab